budgetportal/management/commands/load_departments.py

```diff
@@ -34,39 +34,58 @@ def add_arguments(self, parser):
         parser.add_argument('financial_year', type=str)
         parser.add_argument('sphere', type=str)
         parser.add_argument('filename', type=str)
+        parser.add_argument('update', type=bool, nargs='?', default=False)
 
     def handle(self, *args, **options):
         financial_year = options['financial_year']
         sphere = options['sphere']
         filename = options['filename']
+        update = options.get('update', False)
+
         with open(filename) as csvfile:
             reader = csv.DictReader(csvfile)
             for row in reader:
-                government = Government.objects.get(
-                    sphere__financial_year__slug=financial_year,
-                    sphere__slug=sphere,
-                    slug=slugify(row['government']),
-                )
+                try:
+                    government = Government.objects.get(
+                        sphere__financial_year__slug=financial_year,
+                        sphere__slug=sphere,
+                        slug=slugify(row['government']),
+                    )
+                except Government.DoesNotExist:
+                    print('Missing government: {} {} {}'.format(financial_year, sphere, row['government']))
+                    raise
                 intro = ""
-                if row.get('purpose', False):
-                    intro += "## Vote purpose\n\n%s\n\n" % row['purpose']
-                if row.get('vision', False):
-                    intro += "## Vision\n\n%s\n\n" % row['vision']
-                if row.get('mission', False):
-                    intro += "## Mission\n\n%s\n\n" % row['mission']
-                if row.get('mandate', False):
-                    intro += "## Mandate\n\n%s\n\n" % row['mandate']
-                if row.get('core functions and responsibilities', False):
-                    intro += "## Core functions and responsibilities\n\n%s\n\n" % row['core_functions_and_responsibilities']
+                website_url = None
+                if row.get('intro', False):
```

Comment:
just assign here - don't see any need for append any more?

I think you should make the default None rather than False - then the possible types here are (string or None) rather than (string or boolean)